lib/pry/pry_class.rb

```diff
@@ -1,7 +1,17 @@
 require 'pry/config'
 class Pry
 
-  HOME_RC_FILE = ENV["PRYRC"] || "~/.pryrc"
+  HOME_RC_FILE = proc do
```

Comment:
begin/end should work here instead, but maybe this should be `def self.home_rcfile` ? at least it would be easier to write tests for, and delay querying the filesystem until we have to.